### ISSUE TITLE
The donut (pie chart) 3D offline exporting to PDF failure 

### ISSUE THREAD

**Issue by Basts** [highcharts/highcharts#11501]
The 3D pie chart doesn't render correctly in PDF format while using the offline exporting module. 

#### Live demo with steps to reproduce

A simple [Demo](https://jsfiddle.net/BlackLabel/xv8dt7sn/) which shows the error after exporting to PDF. 

#### Product version

Highcharts JS v7.1.2 (2019-06-03)

#### Affected browser(s)

Tested on the newest versions of Chrome, Safari, and Firefox. 

[chart-1.pdf](https://github.com/highcharts/highcharts/files/3441310/chart-1.pdf)

**Comment by pawelfus** [comment]
Broken since v7.1.0

Bisected to commit f4f33b9534 - which doesn't make sense.. tested config: https://jsfiddle.net/BlackLabel/xv8dt7sn/6/

**Comment by sebastianbochan** [comment]
In 7.0.3 it worked, but I have problem with bisecting which is broken in some commits between 7.0.3 and 7.1.0.

**Comment by nvs2** [comment]
I'm also having this problem with exporting 3D pie charts to PDF in version 8.1.0.  Is there a workaround?

**Comment by sebastianbochan** [comment]
Hi @nvs2,
At this moment we do not have ETA for fixing this issue.

@oysteinmoseng any workaround?

**Comment by oysteinmoseng** [comment]
I don't know of any workaround, from what I can tell in this thread it is still unclear what is causing the issue.

Manual bisecting points to this commit: https://github.com/highcharts/highcharts/commit/35423a9fc7bf841c40e61d3d678dc34f6c8f5ddf - but it is slightly unclear as some of the other commits around this time seem broken. @pawelfus any thoughts? Disabling the inactive state in options did not seem to help.

**Comment by pawelfus** [comment]
It seems issue is not related to inactive state, neither to exporting itself. Standalone SVG with svg2pdf: https://jsfiddle.net/BlackLabel/1odjm0xL/ - in lines 22/25/28 we can find that groups have attrs: `x` and `y`, for example: `<g x="290" y="169">`, which seems to be interpreted by svg2pdf as correct attrs.

In generated SVG we can observe more attributes:
```svg
<g
  data-z-index="31416026.53589793"
  x="290"
  y="169"
  r="159"
  innerR="50"
  start="-1.571"
  end="2.618"
  z="0"
  depth="33.75"
  alpha="45"
  beta="0" 
  center="290,169,318,100"
  stroke="#7cb5ec"
  stroke-width="1"
  stroke-linejoin="round"
>
...
</g>
```

Maybe @Izothep has an idea why those are there? Setting these attrs doesn't make sense for `<g>` (except for `data-z-index`).

**Comment by Izothep** [comment]
@pawelfus I don't think keeping all these parameters in group was intended. Additionally, it looks that adding path to points groups is not working well, it should be definitely refactored.

**Comment by Izothep** [comment]
I have made a check and you are right about the reason of the issue. When setting x and y params as 0 in `SVGRenderer` method responsible for drawing `arc3d`, the problem is not visible anymore.

```
if (typeof params === 'object') {
            params.x = 0;
            params.y = 0;
            ca = suckOutCustom(params);
            if (ca) {
                extend(wrapper.attribs, ca);
                wrapper.setPaths(wrapper.attribs as any);
            }
        }
```

**Comment by anirudhchronus** [comment]
@nvs2 I am having the same problem, pdf export works until 8.0.4 but stops after 8.1.0, Did you find a workaround?

**Comment by KacperMadej** [comment]
The bug was fixed and the current (v9.0.1) code works fine.
